src/main/java/com/minecolonies/coremod/client/render/RenderBipedCitizen.java

```diff
@@ -153,6 +153,18 @@ else if (enumActionOffHand == EnumAction.BOW)
         super.doRender(citizen, d, d1, d2, f, f1);
     }
 
+
```

Comment:
do we need this yet?

Reply:
oh no we dont, just forgot taking it out :)